server/index.js

```diff
@@ -5,4 +5,61 @@ const port = 3003;
 
 app.listen(port, () => {
   console.log(`Server listening at http://localhost:${port}`);
+});
+
+app.use(express.static('public'))
+app.use('/dist', express.static('dist'))
+
+app.patch('/overview/:campId', async (req, res) => {
+  let campId = req.params.campId;
+
+  const mockData = { name: 'Twisselman\'s Glamping by the Pond',
+    location: {
+      name: 'Twisselman Ranch',
+      address: '7645 Cattle Dr, Santa Margarita, CA 93453',
+      numberOfSites: 5
+    },
+    owner: {
+      name: 'Anne B.',
+      imageUrl: 'https://krita-artists.org/uploads/default/original/2X/c/cb096de3604544196cb63799a02405a0e32420bf.jpeg'
+}};
+res.send(mockData);
+});
+
+app.patch('/overview/:campId/location', async (req, res) => {
```

Comment:
You mentioned that you were changing to GET already.

Reply:
yes ma'am!